src/main/java/uk/gov/ch/util/ApiLogger.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.ch.util;
```

Comment:
assume this is from ROE API and already reviewed

Reply:
@AndreKramer7654 Yeah its ripped out of the ROE API :-)